src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -29918,5 +30111,106 @@ class C2 : I(0)
                 Diagnostic(ErrorCode.ERR_UnexpectedArgumentList, "(0)").WithLocation(10, 13)
                 );
         }
+
+        [Theory, WorkItem(44902, "https://github.com/dotnet/roslyn/issues/44902")]
+        [CombinatorialData]
+        public void CrossAssemblySupportingAndNotSupportingCovariantReturns(bool useCompilationReference)
+        {
+            var sourceA =
+@"public record B(int I)
+{
+}
+
+public record C(int I) : B(I);";
+
+            var compA = CreateCompilation(sourceA);
```

Comment:
>CreateCompilation [](http://example.com/codeflow?start=24&length=17)

Since we are not trying to run the code, consider using target framework NetStandard20 here, rather than standard